jaeger_client/reporter.py

```diff
@@ -213,7 +213,7 @@ def _send(self, batch):
         Send batch of spans out via thrift transport. Any exceptions thrown
         will be caught above in the exception handler of _submit().
         """
-        return self.agent.emitBatch(batch)
+        return self.sender.send(batch)
```

Comment:
This is a breaking change it the user still passes the channel. There needs to be an adapter to wrap the old channel and make it look like a sender with `send()` method.

Reply:
Actually, if the user passes the channel, the channel itself is used to generate a default UDPSender (`_create_default_sender(channel)`). 
I dont understand were is here the breaking change

Reply:
+1